### DISCUSSION TITLE
API Key Error

### DISCUSSION THREAD

**Discussion by MPruthi97** [3ru/gpt-translate#70] [Help]
Hello there!

I'm trying to translate content using your action, however, I keep getting this error 'AI_APICallError: The model `gpt-4o` does not exist or you do not have access to it.' even though I'm using a paid OpenAI account. I even added my manager as a collaborator and tried running the action on his computer to no effect.

Please advice.

**Comment by 3ru** [comment]
`/gt ./README.md README.zh_CN.md simplified-chinese`

[a comment by an automated account is not shown]

**Comment by 3ru** [comment]
@MPruthi97 

As you can see, I just gave it a try with `gpt-4o` on my end, and it's working correctly.
Based on the error message you're seeing, it seems that your API key might not have access to the gpt-4o model. While I'm not entirely sure about the specifics, here are a couple of things you can try:

1. Have you prepaid or added credit to [your OpenAI API account](https://platform.openai.com/settings/organization/billing/overview)?
2. Could you try accessing the gpt-4o model directly in the [OpenAI API Playground](https://platform.openai.com/playground/chat?models=gpt-4o)?

**Comment by MPruthi97** [comment]
@3ru I'm trying to run the action by commit. I'll get back to you with answers in sometime as I'm using a key generated through my teammates' paid account so will reach out to him.
